### PR TITLE
Validate OWNERS files

### PR DESCRIPTION
**Pull request by rajatgupta24**
**What type of PR is this?**

For reference: https://github.com/kubernetes/community/issues/6334
/kind cleanup

**What this PR does / why we need it:**

**Which issue(s) this PR fixes:**

Fixes #

**Special notes for your reviewer:**

**Does this PR introduce a user-facing change?:**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -5,6 +5,3 @@ approvers:
 
 reviewers:
 - kubernetes-csi-reviewers
-
-emeritus_reviewers:
-- davidz627
```
Comment (pohly) [inline OWNERS:10]: It feels wrong to remove davidx627 here. Should he be under `emeritus_approvers` (although he never has been an approver)?

Comment (dims) [reply OWNERS:10]: @pohly you can leave it commented out if you wish.

Comment (pohly) [reply OWNERS:10]: @rajatgupta24 please do that.

Perhaps I am missing the purpose of `emeritus_approvers`, but why is that a field that the parser supports? Wouldn't comments serve the same goal (document former members)?

Comment (rajatgupta24) [reply OWNERS:10]: @pohly I commented the section

**Review by pohly — COMMENTED** [review]
/lgtm

/assign @xing-yang 

for approval

**Pull request merged**